cmd/monaco/integrationtest/v1/all_configs_integration_test.go

```diff
@@ -32,14 +34,15 @@ import (
 func TestIntegrationAllConfigs(t *testing.T) {
 	allConfigsFolder := AbsOrPanicFromSlash("test-resources/integration-all-configs/")
 	allConfigsEnvironmentsFile := filepath.Join(allConfigsFolder, "environments.yaml")
+	ctx := context.TODO()
 
-	RunLegacyIntegrationWithCleanup(t, allConfigsFolder, allConfigsEnvironmentsFile, "AllConfigs", func(fs afero.Fs, manifest string) {
+	RunLegacyIntegrationWithCleanup(ctx, t, allConfigsFolder, allConfigsEnvironmentsFile, "AllConfigs", func(fs afero.Fs, manifest string) {
```

Comment:
Should `func(fs afero.Fs, manifest string)` include a context?

Reply:
outdated. Tests will be converted when the new GO version arrives.